File: lua/iota-seed-generator.lua
local key_size = tonumber(arg[1])
local tokenLibrary = "ABCDEFGHIJKLMNOPQRSTUVXYWZ9"
local tokenLibrarySize = string.len(tokenLibrary)
local seed = ""

if key_size < 31 or key_size > 81 then
    print("key_size must be between 31 and 81")
    os.exit(1)
end

for i = 1, key_size do
    local p = math.random(tokenLibrarySize)
    local new_token = string.char(string.byte(tokenLibrary, p))
    seed = string.format("%s%s", seed, new_token)
end

print(seed)

File: go/iota-seed-generator.go
package main

import "fmt"
import (
	"os"
	"strconv"
	"math/rand"
	"time"
)

const tokenLibrary = "ABCDEFGHIJKLMNOPQRSTUVXYWZ9"

func main() {
	keySize, _ := strconv.Atoi(os.Args[1])
	if keySize < 31 || keySize > 81 {
		fmt.Println("keySize must be grater than 30, lower than 82")
		os.Exit(1)
	}
	rand.Seed(time.Now().Unix())
	librarySize := len(tokenLibrary)
	for i:=0; i< keySize; i++ {
		p := rand.Intn(librarySize)
		fmt.Print(string(tokenLibrary[p]))
	}
	fmt.Println()
}
